api/CSSStyleSheet.json

```diff
@@ -294,10 +294,10 @@
             "description": "<code>index</code> is optional",
             "support": {
               "chrome": {
-                "version_added": "60"
+                "version_added": "1"
```

Comment:
Yep, it's optional in Chrome. When an attempt was made to make it non-optional in https://bugs.chromium.org/p/chromium/issues/detail?id=319695, that didn't work out and was reverted. So I think to be pedantic it was non-optional in Chrome 31, but no real need to capture that here I think.